### PR TITLE
add policy blog on nairr rfi and assets

### PR DESCRIPTION
**Pull request by IreneSolaiman**

### CODE REVIEW

**Comment by IreneSolaiman** [comment]
added the policy blog on the NAIRR RFI discussed in #public-policy-updates on Slack! Would love to see this live😍 @julien-c @yjernite

**Comment by yjernite** [comment]
Yay, first PR 🙌 🤗 

My only concern is that that might be a little short for the blog, we usually have longer posts in there. Would it make sense to either:
- put some text from the reply directly in the post text,
- or even better, develop each of the points and add links to current HF efforts?

E.g. mention @mmitchellai as an expert, link to some good model cards especially with [verified evaluations](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english), cite openRAIL as a current effort on the topic of misuse, etc.

**Comment by IreneSolaiman** [comment]
Updated!:))

File: nairr_rfi.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Comments on U.S. National AI Research Resource Interim Report"
```
Comment (yjernite) [inline nairr_rfi.md:2]:
```suggestion
title: "AI Policy @🤗: Comments on U.S. National AI Research Resource Interim Report"
```

**Review by yjernite — APPROVED** [review]
Suggested a title change so we can start building a series, otherwise looks good to me!

File: nairr_rfi.md (hunk 1 above)
Comment (yjernite) [reply nairr_rfi.md:2]: Feel free to change the series name, but we should have something there to make policy posts stand out

**Review by osanseviero — APPROVED** [review]
Thanks for the article! I didn't review the content in depth as @yjernite is already reviewing, but I left some comments regarding the name and the image :)

File: nairr_rfi.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
```
[the remark this answers is not in the document]
Comment (julien-c) [reply nairr_rfi.md:1]: maybe `us-national-ai-research-resource`?

[the remark this answers is not in the document]
Comment (osanseviero) [reply nairr_rfi.md:1]: Maybe report instead of resource

[the remark this answers is not in the document]
Comment (IreneSolaiman) [reply nairr_rfi.md:1]: sadly "resource" is the name of the initiative but i love the `us-national-ai-research-resource` !

**Comment by IreneSolaiman** [comment]
Updated!🤗

File: us-national-ai-research-resource.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Comments on U.S. National AI Research Resource Interim Report"
+---
+
```
Comment (osanseviero) [inline us-national-ai-research-resource.md:4]: We also add a link to the corresponding page in GItHub + a publish date, such as in https://github.com/huggingface/blog/blob/main/accelerate-deepspeed.md?plain=1#L8-L13

Comment (osanseviero) [inline us-national-ai-research-resource.md:4]: You'll also need to update the bottom of https://github.com/huggingface/blog/blob/main/_blog.yml for the blog to show :D

**Review by osanseviero — APPROVED** [review]
Looks great! I left two minor comments and we can merge/deploy afterwards! :fire:

File: us-national-ai-research-resource.md (hunk 3 above)
Comment (IreneSolaiman) [reply us-national-ai-research-resource.md:5]: Done!😍

Comment (IreneSolaiman) [reply us-national-ai-research-resource.md:5]: yes!!

**Pull request merged**